cluster-operator/src/test/java/io/strimzi/operator/cluster/model/EntityOperatorTest.java

```diff
@@ -125,16 +125,22 @@ static Map<String, String> volumeMounts(List<VolumeMount> mounts) {
 
     @ParallelTest
     public void testGenerateDeploymentWithBto() {
-        testGenerateDeployment(false);
+        testGenerateDeployment(false, false);
     }
 
     @ParallelTest
     public void testGenerateDeploymentWithUto() {
-        testGenerateDeployment(true);
+        testGenerateDeployment(true, false);
     }
 
-    private void testGenerateDeployment(boolean useUnidirectionalTopicOperator) {
+    @ParallelTest
+    public void testGenerateDeploymentWithUtoAndCc() {
+        testGenerateDeployment(true, true);
+    }
```

Comment:
It is just test, so if you want to keep this I can live with it. But is it really needed to have these methods which differ just with the name and the true / false?

Reply:
I'm keeping them separated to allow parallel execution.

Reply:
AFAIU we do not use parallel execution, the tag is meaningless at execution time.

Reply:
Ok, now we have one method. 

I understand that ParallelTest annotations are used only when enabled with JUnit configuration. If this is correct, I think they are very confusing as one would expect parallel execution, when there is not. 

I'm not sure why they are not enabled by default, maybe there was some problem. In that case I would suggest to revert back to using the standard Test annotation, unless we are actually using them in some context (our CI?). In the last case, it would be good to add some annotation comment or development documentation.

@see-quick wdyt?

Reply:
> I'm not sure why they are not enabled by default, maybe there was some problem. In that case, I would suggest to revert back to using the standard Test annotation, unless we are actually using them in some context (our CI?

we step back to the sequential model for UTs and ITs because parallel execution introduced a lot of flakiness AFAIK and I did not have much time to investigate the root cause of that flakiness... 

> it would be good to add some annotation comments or development documentation.

sure, that's a good idea.